src/containers/NonBitGoRecovery/Coin/EthereumForm.tsx

```diff
@@ -161,9 +160,11 @@ export function EthereumForm({ onSubmit }: EthereumFormProps) {
           />
         </div>
         <div className="tw-flex tw-flex-col-reverse sm:tw-justify-between sm:tw-flex-row tw-gap-1 tw-mt-4">
-          <Button Variant="secondary" Width="hug" type="reset">
-            Cancel
-          </Button>
+          <Link to="/" className="tw-contents">
+            <Button Variant="secondary" Width="hug" type="reset">
+              Cancel
+            </Button>
+          </Link>
```

Comment:
what you have is not valid HTML

```suggestion
          <Button Tag={Link} to="/" Variant="secondary" Width="hug">
            Cancel
          </Button>
```